concepticondata/conceptlists/Soederholm-2013-420.tsv

```diff
@@ -0,0 +1,421 @@
+Soederholm-2013-420-326	326	Sydän	Heart	1223	HEART	5.38	3.72	5.54	4.96	3.75	3.64	5.1	5.39	5.44	5.5	4.05	3.64	3.9	3.54
+Soederholm-2013-420-327	327	Syksy	Fall	1276	AUTUMN	4.62	3.39	4.66	4.45	3.35	3.55	3.73	4.66	4.72	4.63	3.73	3.32	3.33	4.13
+Soederholm-2013-420-328	328	Sympatia	Sympathy			5.79	3.16	5.89	5.48	3.19	3.06	5.37	5.59	5.99	6.29	3.15	2.97	3.41	2.81
+Soederholm-2013-420-329	329	Syntymä	Birth	2805	BIRTH (ACCOUCHEMENT)	5.47	4.53	5.66	4.91	4.69	4.04	5.11	5.17	5.8	5.71	4.07	4.54	4.74	3.9
+Soederholm-2013-420-330	330	Syrjintä	Discrimination			1.66	5.19	1.65	1.72	5.31	4.81	1.42	1.88	1.54	1.52	4.81	5.37	5.11	5.14
+Soederholm-2013-420-331	331	Sävytys	Tinting			4.37	3.3	4.5	3.97	3.31	3.3	3.67	3.31	3.34	3.33	3.96	4.53	3.95	3.52
+Soederholm-2013-420-332	332	Taakka	Burden	1679	LOAD	2.27	4.33	2.15	2.81	4.38	4.1	2.27	2.11	2.46	2.38	3.64	4.17	4.64	4.5
+Soederholm-2013-420-333	333	Taide	Art			5.5	3.31	5.66	4.97	3.22	3.62	4.45	5.31	5.74	6	3.18	3.2	3.35	3.8
+Soederholm-2013-420-334	334	Taito	Talent			5.48	3.53	5.51	5.35	3.46	3.81	5.18	5.49	5.44	6.13	3.91	3.45	3.48	4.13
+Soederholm-2013-420-335	335	Takiainen	Burdock			2.79	4.11	2.75	3	4.2	3.71	2.64	2.64	2.97	3.25	4.18	4.08	4.22	3.5
+Soederholm-2013-420-336	336	Tapaturma	Accident	4058	ACCIDENT	1.52	5.28	1.47	1.69	5.33	5.1	1.36	1.56	1.55	1.22	4.64	5.29	5.35	5.3
+Soederholm-2013-420-337	337	Tappaja	Killer	3871	MURDERER	1.39	5.74	1.3	1.61	5.81	5.53	1.56	1.44	1.1	1.25	5.56	5.77	5.65	5.83
+Soederholm-2013-420-338	338	Tappio	Defeat	782	DEFEAT	2.16	4.81	2.18	2.1	4.82	4.8	2.23	2.18	2.17	1.79	4.74	4.85	4.62	5
+Soederholm-2013-420-339	339	Tappo	Manslaughter			1.4	5.38	1.35	1.54	5.56	4.85	2	1.41	1.31	1	5.11	5.43	5.43	5.3
+Soederholm-2013-420-340	340	Tarina	Story	2060	STORY	5.41	3.5	5.44	5.31	3.51	3.49	5	5.59	5.29	5.6	3.18	3.4	3.59	3.9
+Soederholm-2013-420-341	341	Tarjoilu	Serving			4.77	3.46	4.76	4.84	3.5	3.26	4.73	4.82	4.67	5.13	3.55	3.36	3.57	3.5
+Soederholm-2013-420-342	342	Tarmo	Vigor			5.28	4.36	5.31	5.1	4.39	4.23	4.55	5.19	5.45	5.75	3.64	4.45	4.34	4.5
+Soederholm-2013-420-343	343	Tarve	Need	643	NEED (NOUN)	4.07	4.04	4.03	4.26	4.13	3.61	4.82	3.82	4.15	5.13	3.82	4.22	3.88	3.75
+Soederholm-2013-420-344	344	Tehokkuus	Efficiency			3.91	4.8	3.96	3.74	4.84	4.67	4	3.86	3.96	3.7	4.45	4.87	4.79	4.8
+Soederholm-2013-420-345	345	Tekniikka	Technique			4.33	3.78	4.13	4.85	3.8	3.73	3.92	4.37	4.34	4.5	3.54	3.83	3.66	4
+Soederholm-2013-420-346	346	Terva	Tar			4.42	3.25	4.35	4.61	3.35	2.99	4.15	4.33	4.7	5.13	3.46	3.22	3.23	3.38
+Soederholm-2013-420-347	347	Tervehdys	Greeting			5.6	3.27	5.71	5.26	3.24	3.36	5.09	5.26	5.91	6.1	3.27	3.29	3.2	3.7
+Soederholm-2013-420-348	348	Terveys	Health	3828	HEALTHINESS	5.88	3.39	5.86	5.97	3.37	3.48	5.73	5.99	5.73	6.13	3.82	3.16	3.59	3.75
+Soederholm-2013-420-349	349	Terävyys	Sharpness			3.69	4.21	3.59	3.99	4.3	3.93	3.67	3.67	3.67	3.9	4	4.42	4.1	3.9
+Soederholm-2013-420-350	350	Teurastus	Slaughter			1.63	5.07	1.54	1.9	5.23	4.57	1.89	1.69	1.54	1.43	4.59	5.34	5.02	4.52
```

Comment:
Inconsistent mappings with respect to N/V, see e.g. [Vergallito-2020-1121-570](https://concepticon.clld.org/values/Vergallito-2020-1121-570) (and not mapping here is of course correct, just noticed the inconsistency).

Reply:
Yes, thank you!